ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -2455,6 +2456,37 @@ def get_healthcheck_version(host):
     return healthcheck_version
 
 
+def wait_for_ipa_to_start(host, timeout=60):
```

Comment:
The default constants.startup_timeout is configured in IPA as 120 seconds, and that's per-service. The CA can be slow to start so 60 seconds may be too short.